app_backend/app/routers/favorite.py

```diff
@@ -0,0 +1,47 @@
+from fastapi import APIRouter, Response, status
+from pydantic import BaseModel
+
+router = APIRouter(prefix="/favorites")
+
+
+class Favorite(BaseModel):
```

Comment:
Missing `uuid` -- how do we know which user to toggle this for?

Reply:
I think we should pass the current user information in a cookie, possibly a JWT token. A user must login in order to change favorite information.

Reply:
In some past projects, we designed it as a Python decorator function, something like

```
@user_logined
@router.post("/toggle-favorite/", status_code=status.HTTP_200_OK)
async def toggle_favorite(resp: Response, favorite: Favorite):
   pass
```

Then the function should have access to the user's information, including the UUID. 

Not sure if there is an existing library for this, though.